lib/OctokitTypes.ts

```diff
@@ -1,3 +1,19 @@
 import { components } from '@octokit/openapi-types/types.d';
 
-export type PullRequest = components['schemas']['pull-request'];
+export type IssueComment = components["schemas"]["issue-comment"];
+export type PullRequest = components['schemas']['pull-request'] & {
+  // Declare extensions for the underlaying type. We can't modify the prototype, unfortunately. 
+  isRenovate(): boolean;
+};
+
+export function addPullRequestExtensions(pr: components['schemas']['pull-request']): PullRequest {  // Adds implementation of declared extensions
+  return {
+    ...pr,
+    ... {
+      isRenovate: function (): boolean {
+        return this.user.login === "renovate[bot]";
+      }
+    }
+  };
+}
```

Comment:
This would be much nicer, but it doesn't work
```suggestion
export type PullRequest = components['schemas']['pull-request'];

PullRequest.prototype.isRenovate = function (): boolean {
  return this.user.login === "renovate[bot]";
}

```